Programs/_testembed.c

```diff
@@ -1567,7 +1567,7 @@ static int test_init_warnoptions(void)
 
     config.dev_mode = 1;
     config.bytes_warning = 1;
-    config.py2x_warning = 1;
+    // config.py2x_warning = 1;
```

Comment:
Should we just delete rather than comment this?

Reply:
Fixed in the first commit below

Reply:
Because I'm old and stupid, can you say "Fixed in commit XYZ" where XYZ is a hash, so github then links to it? It makes reviewing much easier!